src/test/kotlin/com/ecwid/apiclient/v3/rule/nullablepropertyrules/FetchedProductRules.kt

```diff
@@ -99,8 +99,6 @@ val fetchedProductNullablePropertyRules: List<NullablePropertyRule<*, *>> = list
 	IgnoreNullable(FetchedProduct.ProductImage::image800pxUrl),
 	IgnoreNullable(FetchedProduct.ProductImage::imageOriginalUrl),
 	AllowNullable(FetchedProduct.ProductImage::alt),
-	AllowNullable(FetchedProduct.ProductImage.Alt::main),
```

Comment:
The `main` and `translated` properties are still nullable, just because now they are a separate class.